scripts/test/wallet/integration.sh

```diff
@@ -38,7 +38,7 @@ tmux new-session -s $sessionName -d "WALLET_DEBUG=1 scripts/launch/demo-with-wal
 
 # wait until cluster is fully up and running
 echo "Waiting 140 seconds until local cluster is ready..."
-sleep 140s
+sleep 40s
```

Comment:
I dropped this down because it was taking *a long* time to run tests, and i was getting distracted with twitter and friends between runs! can revert